qml/pages/VenueMapOverviewPage.qml

```diff
@@ -110,7 +110,7 @@ BVApp.Page {
                     onClicked: {
                         pageStack.push(Qt.resolvedUrl("VenueDescription.qml"),
                                        {
-                                           restaurant     : model,
+                                           restaurant     : page.model.item(index),
```

Comment:
Reintroduces https://github.com/micuintus/harbour-Berlin-Vegan/issues/185 unfortunately :/

Reply:
Do we know, why `positionSource : page.positionSource` in the next line works correctly?